### PR TITLE
[ru] cs-229-linear-algebra

### PR DESCRIPTION
**Pull request by AlexandrParkhomenko**
web format: [Algebra and Calculus](https://alexandrparkhomenko.github.io/stanford/cs-229/linear-algebra/index.html)

### CODE REVIEW

File: ru/cs-229-linear-algebra.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+**46. diagonal**
+
+&#10230; диагональ
+
+<br>
+
+**47. Singular-value decomposition ― For a given matrix A of dimensions m×n, the singular-value decomposition (SVD) is a factorization technique that guarantees the existence of U m×m unitary, Σ m×n diagonal and V n×n unitary matrices, such that:**
+
+&#10230; Сингулярное разложение ― Для данной матрицы A размеров m×n разложение по сингулярным числам (singular-value decomposition, SVD) представляет собой метод факторизации, который гарантирует существование унитарных матриц U m×m, диагональных Σ m×n и унитарных матриц V n×n, таких что:
```
Comment (geotrush) [inline ru/cs-229-linear-algebra.md:283]:
```suggestion
&#10230; Сингулярное разложение ― Для данной матрицы A размеров m×n разложение по сингулярным числам (singular-value decomposition, SVD) представляет собой метод факторизации, который гарантирует существование следующих матриц: унитарной U m×m, диагональной Σ m×n и унитарной V n×n, таких что:
```

File: ru/cs-229-linear-algebra.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+<br>
+
+**48. Matrix calculus**
+
+&#10230; Матричное исчисление
```
Comment (geotrush) [inline ru/cs-229-linear-algebra.md:289]:
```suggestion
&#10230; Матричное дифференцирование
```

File: ru/cs-229-linear-algebra.md [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+**57. [Matrix calculus, Gradient, Hessian, Operations]**
+
+&#10230; [Матричное исчисление, Градиент, Гессиан, Операции]
+
+<br>
+
+**58. Original authors**
+
+&#10230; Авторы оригинала: Afshine Amidi и Shervine Amidi ― https://github.com/shervinea
```
Comment (geotrush) [inline ru/cs-229-linear-algebra.md:349]:
```suggestion
&#10230; Авторы оригинала: Afshine Amidi и Shervine Amidi ― https://github.com/afshinea и https://github.com/shervinea
```

File: ru/cs-229-linear-algebra.md [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+
+<br>
+
+**59. Translated by X, Y and Z**
+
+&#10230; Российская адаптация: Пархоменко Александр ― https://github.com/AlexandrParkhomenko
```
Comment (geotrush) [inline ru/cs-229-linear-algebra.md:355]:
```suggestion
&#10230; Переведено на русский язык: Пархоменко Александр ― https://github.com/AlexandrParkhomenko
```

File: ru/cs-229-linear-algebra.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+
+<br>
+
+**60. Reviewed by X, Y and Z**
+
+&#10230; Проверено X, Y и Z
```
Comment (geotrush) [inline ru/cs-229-linear-algebra.md:361]:
```suggestion
&#10230; Проверено на русском языке: Труш Георгий (Georgy Trush) ― https://github.com/geotrush
```

**Review by tensorush — CHANGES_REQUESTED** [review]
Great! I've fixed some mistakes.

File: ru/cs-229-linear-algebra.md [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+
+**4. Vector ― We note x∈Rn a vector with n entries, where xi∈R is the ith entry:**
+
+&#10230; Вектор ― Мы обозначаем x∈Rn вектор с n элементами, где xi∈R∈ i-й элемент:
+
+<br>
+
+**5. Matrix ― We note A∈Rm×n a matrix with m rows and n columns, where Ai,j∈R is the entry located in the ith row and jth column:**
+
+&#10230; Матрица ― Мы обозначаем A∈Rm×n матрица с m строками и n столбцами, где Ai,j∈R - запись, расположенная в i-й строке и j-м столбце:
```
[the remark this answers is not in the document]
Comment (AlexandrParkhomenko) [reply ru/cs-229-linear-algebra.md:31]: элемент, расположенный

[the remark this answers is not in the document]
Comment (geotrush) [reply ru/cs-229-linear-algebra.md:31]: @AlexandrParkhomenko Right, I've fixed it.

**Pull request closed without merging**